app/containers/message/styles.js

```diff
@@ -161,5 +161,9 @@ export default StyleSheet.create({
 	},
 	readReceipt: {
 		lineHeight: 20
+	},
+	username: {
+		marginLeft: 8,
+		color: '#333'
```

Comment:
Never use constant colors on `Text`, you need to set themed colors on inline style.